third-party/cmake/BuildLuarocks.cmake

```diff
@@ -41,23 +41,23 @@ endfunction()
 # The luarocks binary location
 set(LUAROCKS_BINARY ${HOSTDEPS_BIN_DIR}/luarocks)
 
-# NOTE: Version must match version of LuaRocks in third-party/CMakeLists.txt
-set(LUAROCKS_VERSION 2.4)
-
 # Arguments for calls to 'luarocks build'
 if(NOT MSVC)
   # In MSVC don't pass the compiler/linker to luarocks, the bundled
   # version already knows, and passing them here breaks the build
   set(LUAROCKS_BUILDARGS CC=${HOSTDEPS_C_COMPILER} LD=${HOSTDEPS_C_COMPILER})
 endif()
 
-if(UNIX OR (MINGW AND CMAKE_CROSSCOMPILING))
+# DEBUG
+list(APPEND LUAROCKS_BUILDARGS --verbose)
+
+if(MINGW OR UNIX OR (MINGW AND CMAKE_CROSSCOMPILING))
```

Comment:
off-topic: I wonder if the `CMAKE_CROSSCOMPILING` cases can be removed since  b98b1168d81a0fd61c073780f0555ab858abdf78 and 55682710a

Reply:
Hmm, good question.. no idea what it does really, and if it would be useful outside of Travis also?

Reply:
commit message in https://github.com/neovim/neovim/commit/55682710a90ab5a70696dcca4567f640c30aab7d implies that cross-compiling doesn't work anyways. @equalsraf was the only one who was cross-compiling. So a quick review of the various `CMAKE_CROSSCOMPILING` sites is all that's needed to decide if it does anything useful.